Lib/test/test_locale.py

```diff
@@ -6,7 +6,7 @@
 import sys
 import codecs
 
-
+@unittest.SkipTest(reason="Library not completed")
```

Comment:
```suggestion
@unittest.SkipTest("TODO: RUSTPYTHON, Library not completed")
```

Reply:
I think we cannot merge this PR as this way. because `setlocale` is a mendatory part of _locale.
If you want to work on setlocale on different PR, could you change `Lib/locale.py` from
```python
try:
    from _locale import *
except ImportError:
```
to only cherry-picking `_locale.localeconv` and not skipping the python fallback part?

Reply:
@youknowone I just commit the implementation for setlocale function! Should I allow the test for locale module after this commit?

Reply:
Sure, that will be perfect